.tekton/pipeline-service-test.yaml

```diff
@@ -357,9 +376,14 @@ spec:
           steps:
             - name: destroy
               image: quay.io/redhat-pipeline-service/ci-runner:main
+              env:
+                - name: KUBECONFIG
               script: |
                 #!/usr/bin/env bash
-                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
```

Comment:
this export command shouldn't be removed, because it's used for switching to the new cluster provisioned by Hypershift.

Reply:
Fixed.